src/Files.App/ResourceDictionaries/StatusCenterStyles.xaml

```diff
@@ -0,0 +1,363 @@
+﻿<!--  Copyright (c) 2023 Files Community. Licensed under the MIT License. See the LICENSE.  -->
+<ResourceDictionary
+	xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	xmlns:converters="using:Files.App.Converters"
+	xmlns:wctconverters="using:CommunityToolkit.WinUI.UI.Converters">
+
+	<ResourceDictionary.ThemeDictionaries>
+		<ResourceDictionary x:Key="Default">
```

Comment:
Is default for light or dark?

Reply:
Default is what Windows uses for the Dark colours

Reply:
Can we rename it to `dark`? It's more explicit and matches what we use throughout the rest of the codebase. As a side note, the order we follow is light, dark, high contrast.

Reply:
> Can we rename it to `dark`? It's more explicit and matches what we use throughout the rest of the codebase. As a side note, the order we follow is light, dark, high contrast.

It is a standard naming that is used by WinUI.  its the colour that is applied if the requested theme is not explicitly Light, or High Contrast

Reply:
Renamed to Dark.

Reply:
Thanks